README.md

```diff
@@ -78,27 +89,74 @@ An instance on Heroku will be accessible through:
 
 ```ruby
 module CuttingEdge
   REFRESH_SCHEDULE = '2h'
 end
 ```
 
-The sample [config.rb](config.rb) contains (within its comments) some examples of constants that you may wish to configure. Here are some descriptions of what the less obvious ones achieve:
+The sample [config.rb](config.rb) contains some examples of constants that you may wish to configure. Here are some descriptions of what the less obvious ones achieve:
 
-* `SECRET_TOKEN`: set a global secret token for administrative purposes. This token is used to configure [hooks](#Refreshing dependency-status-through-git-hooks), and to list [hidden projects](#Hide-Repositories).
+* `SECRET_TOKEN`: set a global secret token for administrative purposes. This token is used to configure [hooks](#Refreshing-dependency-status-through-git-hooks), and to list [hidden projects](#Hide-Repositories).
+* `SERVER_URL`: the link to the app that should be displayed, for instance in emails. Defaults to `"http://#{SERVER_HOST}"`
+* `MAIL_TEMPLATE`: override the [ERB](https://www.stuartellis.name/articles/erb/) template used to render [emails](#Email-Notifications). See [mail.html.erb](lib/cutting_edge/templates/mail.html.erb) for an example on which variables you can use within the template.
 
 ### Email Notifications
 
+CuttingEdge can send email notifications whenever a change in the dependency status of a monitored project is detected. This is disabled by default. Enable it in `config.rb`:
+
+```ruby
+module CuttingEdge
+  MAIL_TO = 'mydeps@mymail.com'  # Default address to send email to. If set to false (=default!), don't send any emails except for repositories that have their 'email:' attribute set in projects.yml
+  MAIL_FROM = "cutting_edge@my_server.com" # From Address used for sending emails.
+end
+```
+
+By default, the app wil try to use an SMTP server on `localhost:25`. Change these settings in your `config.rb` by calling `Mail.defaults`:
+
+```ruby
+# This should be outside the module CuttingEdge
+Mail.defaults do
+  delivery_method :smtp, address: "localhost", port: 1025
+end
+```
+
+See [the mail gem](https://github.com/mikel/mail#sending-an-email) for more information.
+
+You can switch off email notifications for a single project by setting its `email:` key to `false` in [projects.yml](#projects-yml). Alternatively, you can set the `email:` key for a single project to a different address than the default `MAIL_TO`.
+
 ### Adding self-hosted repository servers
 
-You can monitor projects on your own, self-hosted Gitlab or Gitea instances. To do so, you need to tell CuttingEdge about your server by editing `config.rb` as follows:
+You can monitor projects on your own self-hosted Gitlab or Gitea instances. To do so, you need to tell CuttingEdge about your server by editing `config.rb` as follows:
```

Comment:
For future work: should we make `email:` and `MAIL_TO` accept an array of email addresses?

Reply:
I like that idea! Perhaps open a new issue for that?